### PR TITLE
Document mod configuration (CBA Settings)

### PR DESCRIPTION
**Pull request by hbeni**
Enhance wiki install guide on where to look at for configuration of the mod

### CODE REVIEW

**Comment by hbeni** [comment]
fix #806

File: docs/wiki/user/installation.md [hunk 1]
```diff
@@ -20,6 +20,11 @@ title: Installation
 - ACRE2 will try to copy the plugins to your TeamSpeak 3 installation directory. A pop-up will appear describing what the process did.
 - Launch TeamSpeak 3 and enable the ACRE2 plugin in the `Settings -> Plugins` window.
 
+### Configuration
```
Comment (jonpas) [inline docs/wiki/user/installation.md:23]:
```suggestion
### Configuration

```

File: docs/wiki/user/installation.md [hunk 2]
```diff
@@ -20,6 +20,11 @@ title: Installation
...
+You can configure ACRE2 features to your needs. ACRE2 integrates fully into the [CBA_A3 Settings System](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System), accessible via the ingame configuration. This applies to [server side config](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#server-settings) as well to the client side.
```
Comment (jonpas) [inline docs/wiki/user/installation.md]:
```suggestion
You can configure ACRE2 features to your needs. ACRE2 uses the [CBA Settings System](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System), accessible via the in-game configuration. This applies to [server settings](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#server-settings) as well as client settings.
```

File: docs/wiki/user/installation.md [hunk 3]
```diff
@@ -20,6 +20,11 @@ title: Installation
...
+
+Especially for server admins on the shell level, you can get available settings by the configurations [export/import feature](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#export-and-import-function). The options then need to go into [CBA's userconfig](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#userconfig) file.
```
Comment (jonpas) [inline docs/wiki/user/installation.md]:
```suggestion
Server administrators can get available settings using the [export/import features](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#export-and-import-function). The options then need to go into [CBA's userconfig or an addon](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#userconfig).
```

**Review by jonpas — CHANGES_REQUESTED** [review]

File: docs/wiki/user/installation.md (hunk 2 above)
Comment (BrettMayson) [reply docs/wiki/user/installation.md]: Should be `ACRE2 uses the [CBA`, much more natural

File: docs/wiki/user/installation.md (hunk 3 above)
Comment (BrettMayson) [reply docs/wiki/user/installation.md]: `using the [export/import features]`
`need to go into [CBA's userconfig or an addon]`

**Comment by hbeni** [comment]
Thanks jonpas, i added your suggestions

**Review by jonpas — APPROVED** [review]

File: docs/wiki/user/installation.md [hunk 4]
```diff
@@ -20,6 +20,12 @@ title: Installation
 - ACRE2 will try to copy the plugins to your TeamSpeak 3 installation directory. A pop-up will appear describing what the process did.
 - Launch TeamSpeak 3 and enable the ACRE2 plugin in the `Settings -> Plugins` window.
 
+### Configuration
+
+You can configure ACRE2 features to your needs. ACRE2 uses the [CBA Settings System](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System), accessible via the in-game configuration. This applies to [server settings](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#server-settings) as well as client settings.
```
Comment (TheMagnetar) [inline docs/wiki/user/installation.md]:
```suggestion
You can configure ACRE2 features to your needs. ACRE2 uses the [CBA Settings System](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System), accessible via the in-game configuration. This applies to [server](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#server-settings), mission and client settings.
```

File: docs/wiki/user/installation.md [hunk 5]
```diff
@@ -20,6 +20,12 @@ title: Installation
...
+
+Server administrators can get available settings using the [export/import features](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#export-and-import-function). The options then need to go into [CBA's userconfig or an addon](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#userconfig).
```
Comment (TheMagnetar) [inline docs/wiki/user/installation.md]:
```suggestion
Server administrators can get available settings using the [export/import features](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#export-and-import-function). The options then need to go into [CBA's userconfig or an addon](https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#userconfig). Server settings can be configured to override specific mission settings.
```

**Review by TheMagnetar — CHANGES_REQUESTED** [review]

**Comment by jonpas** [comment]
@hbeni can you finish this? I can't commit to your fork.

**Comment by hbeni** [comment]
@jonpas, i did it, and hope its okay now?

**Comment by jonpas** [comment]
You missed this one: https://github.com/IDI-Systems/acre2/pull/808#discussion_r315558172

**Comment by hbeni** [comment]
oops, completed now

**Review by jonpas — APPROVED** [review]

**Comment by jonpas** [comment]
Thank you!

**Review by TheMagnetar — APPROVED** [review]

**Pull request merged**